src/NBitcoin/NetworkOptions.cs

```diff
@@ -31,11 +45,51 @@ public NetworkOptions(uint flags = All)
             this.flags = flags;
         }
 
+        /// <summary>
+        /// Get/Set ProofOfStake flags.
+        /// </summary>
+        public bool IsProofOfStake
```

Comment:
My argument still stands pos is per network not per serialization, so while this is not perfect I would like to make it a stand alone bool and not part of a flag.

Reply:
Done.